demo/js/components/accessibility-demo.js

```diff
@@ -0,0 +1,51 @@
+/*global window:false*/
+/*eslint-disable no-magic-numbers,react/no-multi-comp */
+import React from "react";
+import { VictoryChart } from "Packages/victory-chart/src/index";
+import { VictoryBar } from "Packages/victory-bar/src/index";
+
+export default class App extends React.Component {
+  constructor() {
+    super();
+  }
+
+  render() {
+    const containerStyle = {
+      display: "flex",
+      flexDirection: "row",
+      flexWrap: "wrap",
+      alignItems: "center",
+      justifyContent: "center",
+      maxWidth: "40%",
+      margin: "3%"
+    };
+
+    const chartContainerStyle = {
+      display: "flex",
+      flexDirection: "column",
+      alignItems: "center",
+      width: "100%",
+      height: "100%"
+    };
+
+    return (
+      <div className="demo" style={containerStyle}>
+        <div style={chartContainerStyle}>
+          <h3>Tabbable with aria-labels bar chart</h3>
```

Comment:
Just call this 'Accessible bar chart'? Open to any better labels

Reply:
I'm not picky about how the demos are labeled.